newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/ExpressionAutocompletionsDisplayer.js

```diff
@@ -273,6 +274,8 @@ export default function ExpressionAutocompletionsDisplayer({
   onScroll,
   parameterRenderingService,
 }: Props) {
+  const muiTheme = useTheme();
```

Comment:
Some would say you could use the gdevelopTheme?

Reply:
Well, the alternate background color is used for `gdevelopTheme.list.itemsBackgroundColor` and `gdevelop.searchBar.backgroundColor`.
So I could define a new item in the GDevelop theme that would be something like `gdevelopTheme.palette.background.alternateColor`.

Moreover this `useTheme` was used in only one place (`SelectOption`). So maybe I can also make sure this single use uses the gdevelop theme instead.